extension/src/test/suite/DocumentContainer.test.ts

```diff
@@ -0,0 +1,92 @@
+import * as path from 'path';
+import { Position, Range } from 'vscode';
+import { LispParser } from '../../format/parser';
+import { Sexpression, LispContainer } from '../../format/sexpression';
+import { ReadonlyDocument } from '../../project/readOnlyDocument';
+
+var assert = require('chai').assert;
+let project_path = path.join(__dirname + "\\..\\..\\..\\test_case\\pdfMarkups.lsp");
+
+suite("LispParser.DocumentContainer Tests", function () {	
+	test("Original atomsForest vs DocumentContainer", function () {	
```

Comment:
Thanks for adding the **UNIT** tests!
I suggest we add more MEDIUM tests I mentioned in the comment. For more medium tests, we have more confidence for future code changes.